monkey/monkey_island/cc/models/machine.py

```diff
@@ -26,7 +26,7 @@ class Machine(MutableInfectionMonkeyBaseModel):
     operating_system: Optional[OperatingSystem]
     """The operating system the machine is running"""
 
-    operating_system_version: str
+    operating_system_version: str = Field(default="")
```

Comment:
Why are we not using `default_factory=str`?

Reply:
Strings are immutable. There is only ever one instance of the empty string`""`. Whether we use `str()` or `""`, we'll always get the same object.

```
Python 3.10.4 (main, Jun 29 2022, 12:14:53) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> s1 = str()
>>> s2 = ""
>>> s3 = ""
>>> id(s1)
140408695029808
>>> id(s2)
140408695029808
>>> id(s3)
140408695029808
>>> 
```

Using `""` instead of `str()` seems more explicit to me.